pkg/scheduler/schedule_one.go

```diff
@@ -121,10 +121,11 @@ func (sched *Scheduler) scheduleOne(ctx context.Context) {
 		status := sched.bindingCycle(bindingCycleCtx, state, fwk, scheduleResult, assumedPodInfo, start, podsToActivate)
 		if !status.IsSuccess() {
 			sched.handleBindingCycleError(bindingCycleCtx, state, fwk, assumedPodInfo, start, scheduleResult, status)
+		} else {
```

Comment:
A simpler solution is to add `return` above.

Reply:
done